Fixes for manual_user_install.sh

ensistechnologies opened the pull request:
These two commits fix an incorrect check for the ggwave directory and ensures that the script is run as part of a repo clone so copies of tiles from "stage-core" don't fail if the user has only downloaded the installation script.

silvenshadow committed:
Be sure the script is being run in a copy of the repo

silvenshadow committed:
Fix incorect check for ggwave directory

builderjer reviewed (commented):
So what you are saying, is some users will just download the script instead of the whole repo.  This prevents that?

Bulwagga commented:
If this is what those users want, maybe we also have them consider ovos-installer.

builderjer commented:
Yes, but this does give an option to clone the repo or quit

ensistechnologies commented:
So I added the option for downloading the repo because this happened to me.  I was reccomended to run the script, so I did.  It happily installed almost the entire system then failed on copying mycroft.conf in a non-intuitive way.  I added the clone option for completeness, but in my opinion it should at least fail if it doesn't have the files it needs to copy out of the repo.

silvenshadow committed:
Fix typo.
